huey_monitor/humanize.py

```diff
@@ -75,7 +75,8 @@ def throughput(num, elapsed_sec, suffix='', divisor=1000) -> str:
         if suffix:
             return f'0/{suffix}'
         return '0'
-    rate = num / elapsed_sec
+    # need to prevent ZeroDivisionError:
+    rate = num / max(elapsed_sec, 0.001)
```

Comment:
Hm. Think this is not a good solution, because it calculate with wrong values...

What about:
```
if elapsed_sec==0:
    return '~'
```

Reply:
Or maybe

```
if elapsed_sec==0:
    return '∞'

```

:nerd_face: 

On a more serious note I'd also prefer not to use a hardcoded "minimum" time delta and rather return something else in this rare case.

Reply:
No problem for me Jedie.

The hardcoded value of one millisecond was for me a good approximation
Of minimum timing that could be recorded

Do you want me to do the change in the PR or you do it yourself?

We go for the infinite symbol then?